lib/change_stream.js

```diff
@@ -371,6 +371,8 @@ var processNewChange = function(self, err, change, callback) {
         .close()
         .then(() => (self.cursor = createChangeStreamCursor(self)))
         .then(() => self.next());
+    } else {
+      return self.emit('error', err);
```

Comment:
This works to suppress the first resumable error, but there are frequently back-to-back resumable errors that I need to handle.